docs/iris/src/whatsnew/3.0.2.rst

```diff
@@ -38,6 +38,10 @@ This document explains the changes made to Iris for this release
       developers to easily disable `cirrus-ci`_ tasks. See
       :ref:`skipping Cirrus-CI tasks`. (:pull:`4019`) [``pre-v3.1.0``]
 
+   #. `@pp-mo`_ adjusted the use of :func:`dask.array.from_array` in :func:`iris._lazy_data.as_lazy_data`, to avoid
+      the dask 'test access'.  This makes loading of netcdf files with large numbers variables significantly faster.
+      (:pull:`4135`) [``pre-v3.1.0``]
```

Comment:
@pp-mo Drop the `pre-v3.1.0`. This PR is genuinely targeting `v3.0.x`, and will be merged back to `master`.

The `pre-v3.1.0` contributions must be pulled prematurely from `master` into this feature branch to make it work, which is slightly different :+1:

Reply:
Ok, thanks.  I was just copying from recent submissions + didn't understand what the label was intended for !